plugins/chat/assets/javascripts/discourse/components/chat-message.hbs

```diff
@@ -7,10 +7,30 @@
   {{/if}}
 
   <div
-    {{will-destroy this.teardownChatMessage}}
-    {{did-insert this.decorateCookedMessage}}
+    class={{concat-class
```

Comment:
Yes I guess it's debatable, I kinda like it that way, especially if we have getters for any complex logic.

This one for example:

```
(if
  (or (eq @message.id this.chat.activeMessage.model.id) this.isActive)
  "-active"
)
```

maybe I should write a getter for it

Reply:
Yeah I don't like this level of LISPing in the templates haha

Reply:
I removed some of them, and also did something I wanted which is have well defined boundaries on chat message events, like didInsert, didUpdateMessageId, ... that way we can plug into this in the js file, instead of having more and more of these hooks